Makefile.sp

```diff
@@ -51,8 +52,6 @@ sp-pa11y-install:
 		}
 
 sp-install: $(VENVDIR)
```

Comment:
hi @ru-fu , this option doesn't really do anything after removing the sudo commands.
do we still need to keep it in the makefile?
thanks.